app/policies/article_policy.rb

```diff
@@ -2,28 +2,68 @@
 
 class ArticlePolicy < ApplicationPolicy
   def index?
-    user.root? or user.editor?
+    return true if user.root? or user.editor?
+    return true if user.partner_admin? && user.partners.count.positive?
```

Comment:
i think use "or / and" or "|| / &&", not both

Reply:
fixed